### ISSUE TITLE
Gateway setting is not applied in windows 8.1

### ISSUE THREAD

**Issue by GoogleCodeExporter** [netprofilesmod/netprofilesmod#62]
```
What steps will reproduce the problem?
1.Create a profile for a wireless network with static ip including gateway ip
2.Apply the profile
3.The gateway ip is not applied

What is the expected output? What do you see instead?
The field for the gateway ip in the connection properties (TCP/IPv4) is empty 
after applying the profile, the ip for the gateway is not applied and should be 
added manually

What version of the product are you using? On what operating system?
netprofiles 0.2.3 on windows 8.1

Please provide any additional information below.
Nice little piece of software essential to cover a big issue in windows...
Suggestion for enhancement (should be in a separate issue, sorry) Allow the 
program to be run in systray when minimzed.

```

Original issue reported on code.google.com by `ago.sylv...@gmail.com` on 9 Jun 2014 at 12:34
- Blocked on: #58
